### DISCUSSION TITLE
启动器安装包下载后无法安装

### DISCUSSION THREAD

**Discussion by qvqyl** [natfrp/wiki#333] [使用问题]
![1](https://github.com/natfrp/wiki/assets/95543648/1b297209-89b3-4745-b229-1881a8892f56)
![2](https://github.com/natfrp/wiki/assets/95543648/6ee65f63-ea98-4ade-b053-b0285f3946fd)
![3](https://github.com/natfrp/wiki/assets/95543648/a92f394b-7b8a-449b-a82a-288d820f2ac6)
![4](https://github.com/natfrp/wiki/assets/95543648/207d2587-3b06-48a2-a129-445347138224)

**Comment by fakeboboliu** [comment]
您好，此问题通常为杀毒软件认为我们的软件是具有风险的软件（Riskware）。
杀毒软件的担心是实际的，尽管进行了实名要求限制，我们的服务仍时常出现被滥用的情况。

继续使用的风险主要包括：
 - 被具有恶意的第三方安装到您的电脑上，以在您不知情的情况下访问您的电脑
 - 您自行在不了解程序的情况下不做安全措施地暴露自己电脑的服务，造成安全问题

如果您希望继续使用，请考虑配置杀软的白名单，可以参考 [这个教程](https://doc.natfrp.com/launcher/antivirus.html)

**Reply by qvqyl** [reply]
然后出现了新问题

![微信截图_20230531195846](https://github.com/natfrp/wiki/assets/95543648/2feb884d-1a06-4330-bbde-c7b1f64d586f)

**Reply by fakeboboliu** [reply]
> 然后出现了新问题

您好，这通常是同样的原因，请问您使用的杀毒软件是什么？

**Reply by qvqyl** [reply]
本来是火绒，现在卸了

**Reply by qvqyl** [reply]
然后我给他加进白名单，他还是会出现第一个问题

**Reply by qvqyl** [reply]
然后我就找历史版本，这次可以下载下来文件夹了，但是出现了第二个问题

**Reply by qvqyl** [reply]
找的2.6.0

**Reply by fakeboboliu** [reply]
> 本来是火绒，现在卸了

很抱歉，基于您目前提供的信息，我们无法给出超出上面回复内容的帮助，
如果您想起有什么不对劲的话可以随时补充，我们会根据您提供的信息第一时间进行帮助。

**Reply by fakeboboliu** [reply]
通常来讲在事件查看器中可能存在一些信息，
但是这些事件存在您的隐私信息，只能由您自行检查。

如果您有兴趣自行检查的话，
可以按 Win+R，输入 `eventvwr.msc` ，找到 Windows 日志 - 应用程序。

如果您的系统为 Windows 10 或更高，您的系统自带杀毒可能已介入，请留意右下角的通知。

**Reply by qvqyl** [reply]
这个杀毒软件我已经让程序进入其白名单了
还有，这个日志我应该看些什么呢？

**Reply by fakeboboliu** [reply]
我们注意到您使用 GitHub Discussion 可能有些吃力，您可以考虑加入 QQ 群交流。

**Reply by qvqyl** [reply]
Microsoft-Windows-ModernDeployment-Diagnostics-Provider/ManagementService

来源
DistributedCOM
事件ID
10016
这两个东西在我反复安装的一小时内出现很多次，上面的是一个错误，下面的是一个警告
